### DISCUSSION TITLE
[Feature] Give users the ability to tag a face in photo where face wasn't recognized

### DISCUSSION THREAD

**Discussion by whoindahouse** [immich-app/immich#8751] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Having the ability for users to tag a person in a photo where facial recognition didn't recognize them. Would allow user to select the face in the photo that could then be fed back to the facial recognition model to improve results. 

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by schuhbacca** [comment]
Duplicate of https://github.com/immich-app/immich/discussions/2472#discussioncomment-6850486

**Reply by whoindahouse** [reply]
Sorry; I must have missed that in my search. Thanks for the correction
